packages/vertexai/src/api.ts

```diff
@@ -67,7 +71,10 @@ export function getGenerativeModel(
   requestOptions?: RequestOptions
 ): GenerativeModel {
   if (!modelParams.model) {
-    throw ERROR_FACTORY.create(VertexError.NO_MODEL);
+    throw new VertexAIError(
+      VertexAIErrorCode.NO_MODEL,
+      'Missing model parameter'
```

Comment:
This used to be a lot more detailed - `Must provide a model name. ` +
    `Example: getGenerativeModel({ model: 'my-model-name' })` - we shouldn't lose that. Note that calling `ERROR_FACTORY.create(code)` didn't only include the code but also brought in the message string from `ErrorMap<VertexError>` corresponding to the code.

Reply:
Fixed